tests/ttnn/unit_tests/gtests/ccl/kernels/fabric_erisc_datamover_sender_worker_sender.cpp

```diff
@@ -145,62 +156,40 @@ void kernel_main() {
         uint64_t buffer_address = sender.edm_buffer_addr +
                                   (*sender.buffer_index_ptr * (sender.buffer_size_bytes + sizeof(eth_channel_sync_t)));
         sender.send_payload_blocking_from_address(packet_addr, packet_size);
+        DPRINT << "noc write barrier\n";
```

Comment:
Can we clean up some of the DPRINTs before merging to not pollute the terminal if someone else wants to debug